packages/autocomplete-core/src/propGetters.ts

```diff
@@ -50,7 +50,10 @@ export function getPropGetters<TItem>({
         ].some(contextNode => {
           return (
             contextNode &&
-            (isOrContainsNode(contextNode, event.target as Node) ||
+            (isOrContainsNode(
+              contextNode,
+              ((event as unknown) as TouchEvent).target as Node
```

Comment:
A genuine question. It cannot be `(event as TouchEvent)`?

Reply:
For `TouchEvent` we can indeed because I didn't make them generic in the end (they're meant to be attached on the global object).

However for the others we need to cast them first as `unknown` and then as events because e.g. TypeScript cannot know that `TEvent` and `Event` have about the same signature. Passing by `unknown` makes it possible.

There must be a better way to deal with these event types difference but I'm not sure how.

Reply:
Ah okay. `as` doesn't force casting. And you had to cast it to `unknown` first to cast it to whatever you want.